services/image-analysis/README.md

```diff
@@ -0,0 +1,241 @@
+# Image-analysis Service - Cloud Run Service using Native Spring w/GraalVM
+
+## Build code and publish images
```

Comment:
What environment are you assuming? We usually write instructions for Cloud Shell, so anyone can run them without having to setup an environment